doc: do not allow user to edit and delete builder and bindings when the builder is in use (#95105)
Azure/azure-content: articles/spring-cloud/how-to-enterprise-build-service.md

:::image type="content" source="media/enterprise/how-to-enterprise-build-service/builder-create.png" alt-text="Screenshot of 'Add Builder' pane." lightbox="media/enterprise/how-to-enterprise-build-service/builder-create.png":::

> You can also edit a custom builder when the builder is not used in the a deployment. You can update the buildpacks or the [OS Stack](https://docs.pivotal.io/tanzu-buildpacks/stacks.html), but the builder name is read only.

Reviewer: Add two sections:
1. Use a builder to deploy apps 
if not specifying builder, then use default builder;
if specifying a builder name, then use that builder.
3. Maintain a builder
to edit a builder, xxx
to delete a builder, xxx
Tips for you to update a builder for your apps to consume:
     1. add a new builder with the new configs
     2. redeploy your apps with the new builder.

Reviewer: About Manage service binding section, add a notice on the top to tell the prerequisite to do so:
1. in a new builder
2. in an existing builder without any app deployment linking to it.

```

If the builder isn't specified, the `default` builder will be used.

> The builder is a resource that continuously contributes to your deployments. It provides the latest runtime images and latest buildpacks(including the latest APM agents and so on).
> When you used a builder to deploy the app, this builder and the bindings under this builder is not allowed to edit and delete.
> To apply the changes of the builder please save the configuration as a new builder.
> To delete the builder please remove the deployments used this builder first.

Reviewer 2: ```suggestion
The builder is a resource that continuously contributes to your deployments. It provides the latest runtime images and latest buildpacks (including the latest APM agents and so on).
When you use a builder to deploy the app, this builder and the bindings under this builder aren't allowed to edit and delete.
To apply the changes of the builder, save the configuration as a new builder.
To delete the builder, remove the deployments used by this builder first.
```

    --builder-name <your-builder-name>
```

> **Note: The bindings are not allowed to create, edit and delete when the parent builder is used in a deployment.**

Reviewer 2: ```suggestion
> [!NOTE] 
> The bindings are not allowed to create, edit and delete when the parent builder is used in a deployment.**
```
